src/applications/pensions/PensionsApp.jsx

```diff
@@ -7,10 +7,14 @@ import { useFeatureToggle } from 'platform/utilities/feature-toggles';
 import formConfig from './config/form';
 import { NoFormPage } from './components/NoFormPage';
 import { useBrowserMonitoring } from './hooks/useBrowserMonitoring';
+import { submit } from './config/submit';
 
 export default function PensionEntry({ location, children }) {
   const { useToggleValue, TOGGLE_NAMES } = useFeatureToggle();
   const pensionFormEnabled = useToggleValue(TOGGLE_NAMES.pensionFormEnabled);
+  const pensionModuleEnabled = useToggleValue(
+    TOGGLE_NAMES.pensionModuleEnabled,
+  );
```

Comment:
can put this in submit.rb - eliminate the extra function argument and codeclimate issue.  or make it a helper function and include that in submit.rb